packages/syft/src/syft/service/worker/worker_service.py

```diff
@@ -204,7 +192,17 @@ def delete(
                 f"associated with SyftWorker {uid}"
             )
 
-        if not context.node.in_memory_workers:
+        if IN_KUBERNETES:
+            # Kubernetes will only restart the worker NOT REMOVE IT
+            runner = KubernetesRunner()
+            runner.delete_pod(pod_name=worker.name)
+            return SyftSuccess(
+                message=(
+                    f"Worker id={worker.id} was removed from pool."
+                    "Kubernetes will now re-create the pod."
```

Comment:
I think we should share the message indicating, pool deletion is not allowed in Kubernetes, removing and recreating the pod.

Reply:
fixed